src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,14 +1,11 @@
 <script setup>
-import { ref } from 'vue';
+import { ref,computed } from 'vue';
```

Comment:
```suggestion
import { ref, computed } from 'vue';
```

Reply:
In order to increase the readability of the code, we must take care of its formatting.